polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -22,7 +22,7 @@
     from typing_extensions import TypeGuard
 
 
-T = TypeVar("T", bound=orm.DeclarativeBase)
+T = TypeVar("T")
```

Comment:
Bound is removed here as this doesn't exist as a base in 1.4 so would raise false positives in user's code. This requires extra ignore internal but think that's preferable in terms of API ergonomics.

Note this may not have even been a correct even in 2 only for some patterns.